From 486e4777f375f1ac036e06430059bcac5a45b5d6 Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Wed, 27 May 2015 07:55:59 +0000
Subject: [PATCH] Don't package things we don't build

We are not building the uninstall helper.exe for Windows (#9387) and
omit ICU on Windows as well (#13419). We avoid packaging the respective
binaries then as not doing this breaks our builds.

diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
index 6b15f02..1d9ab5b 100644

Index: browser/installer/package-manifest.in
===================================================================
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -54,9 +54,9 @@
 @RESPATH@/hyphenation/*
 @RESPATH@/browser/@PREF_DIR@/firefox-l10n.js
 @RESPATH@/browser/searchplugins/*
-#ifdef XP_WIN32
-@BINPATH@/uninstall/helper.exe
-#endif
+;#ifdef XP_WIN32
+;@BINPATH@/uninstall/helper.exe
+;#endif
 #ifdef MOZ_UPDATER
 @RESPATH@/update.locale
 @RESPATH@/updater.ini
@@ -111,9 +111,9 @@
 #ifndef MOZ_NATIVE_ICU
 #ifdef MOZ_SHARED_ICU
 #ifdef XP_WIN
-@BINPATH@/icudt@MOZ_ICU_DBG_SUFFIX@@MOZ_ICU_VERSION@.dll
-@BINPATH@/icuin@MOZ_ICU_DBG_SUFFIX@@MOZ_ICU_VERSION@.dll
-@BINPATH@/icuuc@MOZ_ICU_DBG_SUFFIX@@MOZ_ICU_VERSION@.dll
+;@BINPATH@/icudt@MOZ_ICU_DBG_SUFFIX@@MOZ_ICU_VERSION@.dll
+;@BINPATH@/icuin@MOZ_ICU_DBG_SUFFIX@@MOZ_ICU_VERSION@.dll
+;@BINPATH@/icuuc@MOZ_ICU_DBG_SUFFIX@@MOZ_ICU_VERSION@.dll
 #elif defined(XP_MACOSX)
 @BINPATH@/libicudata.@MOZ_ICU_VERSION@.dylib
 @BINPATH@/libicui18n.@MOZ_ICU_VERSION@.dylib
